File: java/algorithms/src/main/java/com/tatumalenko/algorithms/java/leetcode/HolidayDutyFree.java
package com.tatumalenko.algorithms.java.leetcode;

public class HolidayDutyFree {

    public static int dutyFree(final int normPrice, final int discount, final int hol) {
        var n = 1;

        while (normPrice * (discount / 100.0) * n <= hol) {
            n += 1;
        }

        return n;
    }

    public static void main(final String[] args) {
        System.out.println(dutyFree(17, 10, 500));
    }

}

File: java/algorithms/src/main/java/com/tatumalenko/algorithms/leetcode/holidayDutyFree.kt
package com.tatumalenko.algorithms.leetcode

import java.lang.Math.floor


class HolidayDutyFreeKt {
    fun dutyFree(normPrice: Int, discount: Int, hol: Int): Int {
        var n = 1

        while (normPrice * discount / 100 * n <= hol) {
            n += 1
        }
        floor(1.2)
        return n
    }

    fun main() {
        System.out.println(dutyFree(17, 10, 500))
    }
}

